helmfile.d/values/_helpers.tpl

```diff
@@ -83,3 +82,7 @@ after the separator.
 {{- define "gen.reg-rep-img" -}}
 {{ join "/" (compact (list .registry .repository .image)) }}
 {{- end }}
+
+{{- define "gen.container_uri" -}}
```

Comment:
Note, by convention template names are usually kebab-case, but I missed that on the parse template so I can let this slip and let it be reworked later.